libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentAlert.kt

```diff
@@ -0,0 +1,251 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+import com.mapbox.geojson.Point
+import java.util.Date
+
+class IncidentAlert private constructor(
```

Comment:
Could you add a test that extends the `BuilderTest`?

Reply:
in progress 🚀

Reply:
done